spec/BowlingSpec.js

```diff
@@ -0,0 +1,114 @@
+describe("Bowling", function() {
+  var bowling,
+      playerOne,
+      playerTwo;
+
+  beforeEach(function() {
+    bowling = new Bowling();
+    playerOne = "Tom";
+    playerTwo = "Alex";
+  });
+
+  it(" should hae a list of players", function() {
```

Comment:
Is this necessarily what your test does?

Reply:
no I guess you've got me here, I guess this test is that it should be able to hold players in a list before any players are added